rotkehlchen/tests/utils/premium.py

```diff
@@ -222,3 +224,9 @@ def assert_db_got_replaced(rotkehlchen_instance: Rotkehlchen, username: str):
 
     assert main_db_exists
     assert backup_db_exists
+
+
+def asset_database_wasnt_replaced(rotkehlchen_instance: Rotkehlchen):
```

Comment:
```suggestion
def assert_database_wasnt_replaced(rotkehlchen_instance: Rotkehlchen):
```

It's also an overkill to create a function to check one thing if that function is not used anywhere else. Just move those 2 checks inside the test code and lose the function.